style.css

```diff
@@ -105,95 +110,117 @@ figcaption {
     padding: 5px 0;
 }
 
+.image-bg-black {
+    background-color: #000;
+}
 
-/** BASE CLASSES */
+.image-bg-black p.caption {
+    background-color: transparent;
+    color: #fff;
+}
 
-.annotate{
-  position: relative;
+.image-bg-black .container.image p.caption {
+    padding: 5px 10px 15px;
```

Comment:
Even padding so that the caption has the same negative space on the. top as it does on the bottom